docs/config-flatcar-v1_0.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Flatcar v1.0.0
+parent: Configuration specifications
+nav_order: 99
+---
+
+# Flatcar Specification v1.0.0
+
+The Flatcar configuration is a YAML document conforming to the following specification, with **_italicized_** entries being optional:
+
+* **variant** (string): used to differentiate configs for different operating systems. Must be `flatcar` for this specification.
+* **version** (string): the semantic version of the spec for this document. This document is for version `1.0.0` and generates Ignition configs with version `3.3.0`.
```

Comment:
It looks like this doc was copied from a 3.2.0-based spec doc, and doesn't include 3.3.0 changes.